test/test_utils/ecs.py

```diff
@@ -807,7 +807,10 @@ def setup_ecs_inference_service(
         model_base_path = get_tensorflow_model_base_path(docker_image_uri)
         _, image_framework_version = get_framework_and_version_from_tag(docker_image_uri)
         if Version(image_framework_version) in SpecifierSet(">=2.7"):
-            arguments_dict["container_command"] = [build_tensorflow_inference_command_tf27_and_above(model_name)]
+            entrypoint = "/usr/bin/tf_serving_entrypoint.sh"
+            if processor == "neuron":
+                entrypoint = "/usr/local/bin/entrypoint.sh"
+            arguments_dict["container_command"] = [build_tensorflow_inference_command_tf27_and_above(model_name, entrypoint)]
```

Comment:
This is the same as the last two comments as it seems we are adding more logic where the main issue can be solved within the dockerfile.

Reply:
Yes. I understand but neuron DLC's have always been using entrypoint.sh.

Reply:
If this is the case, why is this being added now, versus earlier, if this has always been done?

Reply:
The code added was specific to tf27 and above and the last neuron DLC release was for tf25.